builtin/providers/aws/resource_aws_security_group.go

```diff
@@ -359,7 +359,10 @@ func resourceAwsSecurityGroupDelete(d *schema.ResourceData, meta interface{}) er
 
 	log.Printf("[DEBUG] Security Group destroy: %v", d.Id())
 
-	deleteLingeringLambdaENIs(conn, d)
+	err := deleteLingeringLambdaENIs(conn, d)
+	if err != nil {
```

Comment:
Since only the err is used here can we use the `if err := ...; err != nil {` style here?